sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/validation/CapabilityValidator.java

```diff
@@ -22,14 +19,10 @@ public void validate(ServiceSpec serviceSpec) throws CapabilityValidationExcepti
     }
 
     private void validateGpuPolicy(ServiceSpec serviceSpec) throws CapabilityValidationException {
-        try {
-            if (DefaultService.serviceSpecRequestsGpuResources(serviceSpec)
-                    && !Capabilities.getInstance().supportsGpuResource()) {
-                throw new CapabilityValidationException(
-                        "This cluster's DC/OS version does not support setting GPU_RESOURCE");
-            }
-        } catch (IOException e) {
-            throw new CapabilityValidationException("Failed to determine capabilities: " + e.getMessage());
+        if (DefaultService.serviceSpecRequestsGpuResources(serviceSpec)
```

Comment:
The diff and the one below look like git is just not diffing well.  This should be just removing the try/catch.  Can another set of eyes validate, please.